.github/workflows/ci-workflow.yml

```diff
@@ -90,70 +117,17 @@ jobs:
       env:
         VERSIONED_MODE: ${{ github.ref == 'refs/heads/main' && '--minor' || '--major' }}
         JOBS: 4 # 2 per CPU seems to be the sweet spot in GHA (July 2022)
-
-  unit-async-local:
```

Comment:
we need to restore these jobs. the issue now is these are getting run as steps within existing unit/integration/versioned and cannot take advantage of parallelism as much. also we now run the risk if the async local versioned tests fail we have to re run the versioned tests and async local versioned tests

Reply:
You can see what I describe [here](https://github.com/newrelic/node-newrelic/actions/runs/3324164104/attempts/1)

Reply:
having the unit, integration, versioned tests in one job will make them run consecutively.  I think we need a job per type of async local test.  I know it's not great but we should think about how to consolidate this logic eventually.  Also wrt lack of versioned coverage on main, I forgot to capture but I envisioned either a daily job or a job that runs on pushes to main that runs only versioned tests with --major to collect coverage

Reply:
also we could probably do away with the unit/integration tests with the async local flag and just run the versioned tests, that was carry over from when Michael PRd it.  We have specific tests for async local storage that always run regardless of the job type